Libplanet.Net.Tests/SwarmTest.cs

```diff
@@ -1792,6 +1793,129 @@ public async Task LastMessageTimestamp()
             }
         }
 
+        [RetryFact(10, Timeout = Timeout)]
+        public async Task RegulateGetBlocksMsg()
+        {
+            var options = new SwarmOptions
+            {
+                TaskRegulationOptions =
+                {
+                    MaxTransferBlocksTaskCount = 3,
+                },
+            };
+            var apvOptions = new AppProtocolVersionOptions();
+
+            var key = new PrivateKey();
+            Swarm<DumbAction> swarm = await CreateSwarm(
+                    options: options,
+                    appProtocolVersionOptions: apvOptions)
+                .ConfigureAwait(false);
+            NetMQTransport transport = await NetMQTransport.Create(
+                key,
+                apvOptions,
+                new HostOptions("localhost", Enumerable.Empty<IceServer>()));
+
+            try
+            {
+                await StartAsync(swarm);
+                _ = transport.StartAsync();
+                await transport.WaitForRunningAsync();
+                var tasks = new List<Task>();
+                var content = new GetBlocksMsg(new[] { swarm.BlockChain.Genesis.Hash });
+                for (int i = 0; i < 5; i++)
+                {
+                    tasks.Add(
+                        transport.SendMessageAsync(
+                            swarm.AsPeer,
+                            content,
+                            TimeSpan.FromMilliseconds(1000),
+                            1,
+                            false,
+                            default));
```

Comment:
```suggestion
                        Task.Run(async () => await transport.SendMessageAsync(
                            swarm.AsPeer,
                            content,
                            TimeSpan.FromMilliseconds(1000),
                            1,
                            false,
                            default)));
```

`Task.Run()` may be needed especially in xunit test cases (holding SynchronizationContext)